server/static/tokenize.js

```diff
@@ -0,0 +1,3 @@
+tokenize = function(args) {
+    console.log(args);
```

Comment:
TODO: Make sure this gets changed

Reply:
This is our test verification server, will not be part of production deployment.